examples/scotty-examples.cabal

```diff
@@ -1,159 +1,126 @@
-Name:                scotty-examples
-Version:             0.1
-Synopsis:            scotty examples
-Homepage:            https://github.com/scotty-web/scotty
-Bug-reports:         https://github.com/scotty-web/scotty/issues
-License:             BSD3
-License-file:        LICENSE
-Author:              Andrew Farmer <xichekolas@gmail.com>
-Maintainer:          Andrew Farmer <xichekolas@gmail.com>
-Copyright:           (c) 2012-Present Andrew Farmer
-Category:            Web
-Stability:           experimental
-Build-type:          Simple
-Cabal-version:       >= 1.10
-Description:         Example programs using @scotty@
-tested-with:         GHC == 7.6.3
-                   , GHC == 7.8.4
-                   , GHC == 7.10.3
-                   , GHC == 8.0.2
-                   , GHC == 8.2.2
-                   , GHC == 8.4.4
-                   , GHC == 8.6.5
-                   , GHC == 8.8.4
-                   , GHC == 8.10.4
-                   , GHC == 9.0.1
+cabal-version: 3.0
+
+name: scotty-examples
+version: 0.13
+synopsis: scotty examples
+homepage: https://github.com/scotty-web/scotty
+bug-reports: https://github.com/scotty-web/scotty/issues
+license: BSD-3-Clause
+license-file: LICENSE
+author: Andrew Farmer <xichekolas@gmail.com>
+maintainer: Daniel Brice <danielbrice@gmail.com>
+copyright: (c) 2012-Present Andrew Farmer
+category: Web
+stability: experimental
+build-type: Simple
+description: Example programs using @scotty@
+
+tested-with:
+      GHC == 9.4.2
+    , GHC == 9.2.4
+    , GHC == 9.0.2
+    , GHC == 8.10.7
+
+source-repository head
+  type: git
+  location: https://github.com/scotty-web/scotty
+  subdir: examples
+
+common examples-common
```

Comment:
I put Scotty and GHC boot libs in a common `build-depends`. This adds some spurious `build-depends` to some of the examples, but I figure it's a moot point as there's not really going to be a Haskell distribution that doesn't include the GHC boot libs already.